zeyple/zeyple.py

```diff
@@ -83,6 +89,78 @@ def gpg(self):
 
         return ctx
 
+    def _validate_dnssec(self, domain):
+        # if force_dnssec is enabled go ahead
+        if self.config.has_option('zeyple', 'force_dnssec') and \
+            self.config.getboolean('zeyple', 'force_dnssec'):
+
+            dnssec_check = urllib.urlopen("http://portfolio.sidnlabs.nl:8080/check/" + domain).read().split(',')
```

Comment:
Two things here:

 * I'm not a big fan of adding dependency to some third party website that could go down, especially on the `8080` port that could be blocked in some context. I understand that this an optional feature, I just don't want to promote some proprietary hosted service. I'm not sure if it's the case of this service. What do you think @infertux 
  * Other thing, you should (if we want to keep this feature) escape the domain (`urllib.parse.quote` in python 3) 

And what if this API call fail, do we just get an error? What should happen? Is that expected?